/mnt/phd/jihu/VaxDemandDistance/utils/evaluate_model.py:63: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  selected_locations['id'] = range(selected_locations.shape[0])
Start evaluate stage for setting _10000_5_4q_mnl...

Evaluating random order FCFS with Chain type: Dollar; Model: MNL; M = 5, K = 10000, R = None.
 Results stored at /export/storage_covidvaccine/Result/MNL/M5_K10000_4q/Dollar

Import optimization solution from file /export/storage_covidvaccine/Result/MNL/M5_K10000_4q/Dollar/vaccinated/z_total_10000_5_4q_mnl

Distdf not computed for current setting, start computing...

Start computing distdf...

Start constructing blocks...

Start assignments...

Computed abepsilon in: 1.305 seconds.
Assigning individuals...
time3 - time2: 28.567
time4 - time3: 283.312
Number of full locations: 428  out of  828
Finished evaluate stage for setting _10000_5_4q_mnl!

